packages/wdio-browserstack-service/src/service.ts

```diff
@@ -670,12 +751,16 @@ export default class BrowserstackService implements Services.ServiceInstance {
             name = `${pre}${test.parent}${post}`
         }
 
-        this._percyHandler?._setSessionName(name)
+        if (!BrowserstackCLI.getInstance().isRunning()) {
+            this._percyHandler?._setSessionName(name)
+        }
 
-        if (name !== this._fullTitle) {
+        if (name !== this._fullTitle && !BrowserstackCLI.getInstance().isRunning()) {
```

Comment:
What case is this, where we don't want to update title in CLI flow?

Reply:
sessionName should not be marked from this condition if CLI flow is there